webpack/scenes/ContainerImages/Synced/Details/ManifestDetails.js

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect } from 'react';
+import { useParams, useHistory, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Title,
+  PageSection,
+  Grid,
+  GridItem,
+  TextContent,
+  Text,
+  TextVariants,
+  ClipboardCopy,
+  Label,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import Loading from 'foremanReact/components/Loading';
+import LongDateTime from 'foremanReact/components/common/dates/LongDateTime';
+import EmptyStateMessage from '../../../../components/Table/EmptyStateMessage';
+import getDockerTagDetails from './ManifestDetailsActions';
+import {
+  selectDockerTagDetails,
+  selectDockerTagDetailStatus,
+  selectDockerTagDetailError,
+} from './ManifestDetailsSelectors';
+
+const ManifestDetails = () => {
+  const { id } = useParams();
+  const tagId = Number(id);
+  const history = useHistory();
+  const location = useLocation();
+  const dispatch = useDispatch();
+
+  const searchParams = new URLSearchParams(location.search);
+  const manifestId = searchParams.get('manifest');
+
+  useEffect(() => {
+    dispatch(getDockerTagDetails(tagId));
+  }, [dispatch, tagId]);
+
+  const manifestData = useSelector(state =>
+    selectDockerTagDetails(state, tagId), shallowEqual) || {};
+  const status = useSelector(state =>
+    selectDockerTagDetailStatus(state, tagId));
+  const error = useSelector(state =>
+    selectDockerTagDetailError(state, tagId));
+
+  const getManifest = () => {
+    if (!manifestData) return null;
+    return manifestData.manifest || manifestData.manifest_schema2 || manifestData.manifest_schema1;
+  };
+
+  const getDisplayManifest = () => {
+    const parentManifest = getManifest();
+
+    if (!manifestId || !parentManifest) {
+      return parentManifest;
+    }
+
+    if (parentManifest.manifest_type === 'list' && parentManifest.manifests) {
+      const childManifestId = parseInt(manifestId, 10);
+      return parentManifest.manifests.find(m => m.id === childManifestId);
+    }
+
+    return parentManifest;
+  };
+
+  const getShortDigest = (digest) => {
```

Comment:
The shortening here matches what I see in quay: https://quay.io/repository/centos-bootc/centos-bootc?tab=tags